src/components/tooltip/Tooltip.tsx

```diff
@@ -357,7 +370,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
                 role="presentation"
             >
                 <div
-                    role={theme === TooltipTheme.ERROR ? undefined : 'tooltip'}
+                    role={theme === TooltipTheme.ERROR ? undefined : role}
```

Comment:
might be able to combine the `theme === TooltipTheme.ERROR` part into the new `role` variable. although could also keep it like this if it's easier to understand

Reply:
I was having a hard time finding a cleaner way to do this, trying to avoid nested ternary and if statements and lets... do you have any suggestion to make it cleaner?

Reply:
the cleanest way i could think of was
```
let role;
if (isFocusedTrapped) {
    role = 'dialog';
}
else if (theme !== TooltipTheme.ERROR) {
    role = 'tooltip';
}
```
but i don't know if this is an improvement

Reply:
Yeah, I'm indifferent! I suppose it's nice it's all in one place